statement.go

```diff
@@ -918,23 +918,27 @@ func runInNewOrExistRwTxForExplain(session *Session, f func() (affected int64, p
 }
 
 type BeginRwStatement struct {
-	Priority pb.RequestOptions_Priority
+	Priority       pb.RequestOptions_Priority
+	TransactionTag string
```

Comment:
Similarly, this should be simply `Tag string` since this tag is also used as a request tag.

Reply:
fixed.